tests/ttnn/unit_tests/gtests/test_matmul_benchmark.cpp

```diff
@@ -268,7 +268,8 @@ TEST_P(Matmul2DHostPerfTestFixture, Matmul2DHostPerfTest) {
         per_core_M,
         per_core_N,
         /*transpose_mcast=*/false,
-        /*fused_activation=*/std::nullopt};
+        /*fused_activation=*/std::nullopt,
```

Comment:
would be nice if we could have a test where throttling is actually used, and verifies that version of the op with throttling is significatnly slower than w/o it.